.travis.yml

```diff
@@ -11,4 +13,4 @@ env:
 before_script: flake8 civis
 script:
     - py.test --cov civis
-    - sphinx-build -b html -n docs/source/ docs/build/
+    - if [[ $TRAVIS_PYTHON_VERSION == 3* ]] ; then sphinx-build -b html -n docs/source/ docs/build/ ; fi
```

Comment:
This is okay for now.  I believe there is an outstanding bug that docs only build in certain versions of Python 3.  We'll sort that out later since we're hosting docs.